Public/Sdk/Public/Managed/Tools/ResGen.Lite/resgen.lite.dsc

```diff
@@ -23,17 +23,17 @@ namespace Tool {
                 importFrom("System.Threading.Tasks.Extensions").pkg,
 
                 ] : []),
-            importFrom("Microsoft.CodeAnalysis.Common").pkg,
-            importFrom("Microsoft.CodeAnalysis.CSharp").pkg,
-            importFrom("Microsoft.CodeAnalysis.CSharp.Workspaces").pkg,
-            importFrom("Microsoft.CodeAnalysis.Workspaces.Common").pkg,
+            importFrom("Microsoft.CodeAnalysis.Common").withQualifier({ targetFramework: "netstandard2.0" }).pkg,
+            importFrom("Microsoft.CodeAnalysis.CSharp").withQualifier({ targetFramework: "netstandard2.0" }).pkg,
+            importFrom("Microsoft.CodeAnalysis.CSharp.Workspaces").withQualifier({ targetFramework: "netstandard2.0" }).pkg,
+            importFrom("Microsoft.CodeAnalysis.Workspaces.Common").withQualifier({ targetFramework: "netstandard2.0" }).pkg,
 
             // There is a gap in our nuget integration so temporarily adding the dependencies manually

```

Comment:
>e is a gap in our nuget integration so temporarily ad [](start = 19, length = 53)

This comment is actually wrong :)
By design we do NOT pass the transitive nuget references where msbuild does do the transitive. In my view those are training wheels that work fine for small codebases but in larger codebases wiht say > 100 projects thins bloat and escalate fast. It also means that a nuget packages dependencies are part of their public signature so a high chance changes here will be breaking changes. Asking consumers to manually specify the dependencies gives a more robust experience with less version issues and less breaks

I agree that for some packages it would be super useful to have the compiler take the entire transitive dependencies, but I was thinking of having a modifier for that on the reference. We just haven't gotten to that.

Reply:
Good thought, once we switch to Nuget assemblies and remove our hand written parser, this is a feature we should add!